packages/opentelemetry-exporter-jaeger/src/types.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+
+/**
+ * Options for Jaeger configuration
+ */
+export interface ExporterConfig {
+  logger?: types.Logger;
+  serviceName: string;
+  tags?: Tag[];
+  host?: string; // default: 'localhost'
+  port?: number; // default: 6832
+  maxPacketSize?: number; // default: 65000
+}
+
+// tslint:disable-next-line:variable-name
+export const UDPSender = require('jaeger-client/dist/src/reporters/udp_sender')
```

Comment:
Would this work: `export {default as UDPSender} from 'jaeger-client/dist/src/reporters/udp_sender'` ?

I may totally be missing something here, but if the above doesn't work, what would you think of a short comment to explain why this is needed

Reply:
This is required as `jaeger-client` types does not expose the `thrift`, `udp_sender` etc. modules.

Reply:
OK, cool. Could you add that as a comment above to clarify?